src/conversions.rs

```diff
@@ -646,6 +646,15 @@ impl ToJSValConvertible for *mut JSObject {
     }
 }
 
+// https://heycam.github.io/webidl/#es-object
+impl ToJSValConvertible for ptr::NonNull<JSObject> {
+    #[inline]
+    unsafe fn to_jsval(&self, cx: *mut JSContext, rval: MutableHandleValue) {
+        rval.set(ObjectOrNullValue(self.get()));
```

Comment:
This can probably use `ObjectValue` and `maybe_wrap_object_value`.

Reply:
Done. Indeed, that’s what the equivalent impl in Servo does.